veggieseasons/lib/widgets/settings_item.dart

```diff
@@ -132,20 +132,40 @@ class SettingsItemState extends State<SettingsItem> {
                           crossAxisAlignment: CrossAxisAlignment.start,
                           children: <Widget>[
                             SizedBox(height: 8.5),
-                            Text(widget.label),
+                            Text(
+                              widget.label,
```

Comment:
Consider making text styles in `styles.dart` for these Text widgets as well. They were originally copied into the project from another source, which is why they're set up slightly differently. There's no reason they can't be the same as the others, though.